src/client/game/board.ts

```diff
@@ -538,6 +542,29 @@ function getTile(participantID: string): HTMLDivElement {
   return participantTile;
 }
 
+export function updateAudioLevel(participantID: string, audioLevel: number) {
```

Comment:
Hmm good point; I am going to avoid calculating the opacity at all if the audio level is 0, and just set opacity to 0 as well in that case (I did benchmark the performance impact of setting a CSS property to the same value with a check of prior value vs without, and confirmed that checking prior value seems slower than just overwriting the identical value). 

For audio levels other than 0 I think adding an object of prior audio levels may be a bit overkill for this implementation (to keep the diff simple and focus on showcasing the feature), since the opacity calculation isn't that heavy anyway.